### PR TITLE
Adding patch file support for Open Embedded.

### PR DESCRIPTION
**Pull request by allenh1**
This includes fies in the `files` directory as patches within the generated recipe text.

connects to #129

### CODE REVIEW

**Comment by allenh1** [comment]
current result is allenh1/meta-ros#28

File: superflore/generators/bitbake/run.py [hunk 1]
```diff
@@ -99,10 +99,11 @@ def main():
             )
         else:
             pr_comment = pr_comment or (
-                'Superflore yocto generator began regeneration of package(s)' +
-                ' %s from ROS distro %s from allenh1\'s fork of Meta-ROS ' +
-                'from commit %s.' % (
+                'Superflore yocto generator began regeneration of package(s)'
+                ' %s from ROS distro %s from allenh1\'s fork of Meta-ROS'
```
Comment (tfoote) [inline superflore/generators/bitbake/run.py]: This is missing a parameterization based on #128

File: superflore/generators/bitbake/yocto_recipe.py [hunk 2]
```diff
@@ -173,5 +174,17 @@ def get_recipe_text(self, distributor, license_text):
         ret += 'SRC_URI[sha256sum] = "' + self.src_sha256 + '"\n'
         ret += 'S = "${WORKDIR}/'
         ret += self.get_src_location() + '"\n\n'
+        # Check for patches
+        if self.patch_files:
+            ret += 'SRC_URI += "\\\n'
+            ret += ' \\\n'.join(self.patch_files) + '"\n\n'
+        if self.name == 'catkin':
```
Comment (tfoote) [inline superflore/generators/bitbake/yocto_recipe.py]: Can these be integrated into patches?

Comment (allenh1) [reply superflore/generators/bitbake/yocto_recipe.py]: In the normal sense, no. I think I could, however, just detect an existing `.inc` file and include it.

File: superflore/generators/bitbake/run.py (hunk 1 above)
Comment (allenh1) [reply superflore/generators/bitbake/run.py]: Honestly, I think the `allenh1` part can be removed.

**Pull request merged**